products/unity.md

```diff
@@ -15,6 +15,13 @@ changelogTemplate: "https://unity.com/releases/editor/whats-new/{{'__LATEST__'|s
 
 releases:
 -   releaseCycle: "6"
```

Comment:
```suggestion
-   releaseCycle: "6.1"
```

Reply:
Not even unity itself calls it 6.1 , they are under `Unity 6` name

Reply:
Except on https://unity.com/releases/unity-6/support, https://unity.com/blog/unity-6-1-is-now-available, https://docs.unity3d.com/Manual/index.html...

Furthermore `releaseCycle` cannot be the same between two releases: `releaseCycle` is used as an identifier in API.